internal/pkgecosystem/pypi.go

```diff
@@ -31,14 +31,8 @@ func getPyPILatest(pkg string) string {
 }
 
 var PyPIPackageManager = PkgManager{
-	Name:  "pypi",
-	Image: "gcr.io/ossf-malware-analysis/python",
-	CommandFmt: func(pkg, ver string) string {
-		if ver != "" {
-			return fmt.Sprintf("analyze.py %s==%s", pkg, ver)
-		}
-
-		return fmt.Sprintf("analyze.py %s", pkg)
-	},
-	GetLatest: getPyPILatest,
+	Name:        "pypi",
+	Image:       "gcr.io/ossf-malware-analysis/python",
+	CommandPath: "analyze.py",
```

Comment:
Another way we could simplify all of this is to just use Docker's ENTRYPOINT to specify this. Then we just invoke the sandbox like:

```
podman run gcr.io/ossf-malware-analysis/python --local BLAH --version BLAH
```

Reply:
Great idea. ENTRYPOINT is exactly the problem this is solving.

Reply:
Done.